cli/command/container/opts.go

```diff
@@ -381,32 +381,28 @@ func parse(flags *pflag.FlagSet, copts *containerOptions, serverOS string) (*con
 	publishOpts := copts.publish.GetAll()
 	var ports map[nat.Port]struct{}
 	var portBindings map[nat.Port][]nat.PortBinding
+	var convertedOpts []string
```

Comment:
nit: use a `var` block ?

Reply:
Ah, yes, I'll update once the other PR is merged (and this one rebased)